uint16 trajtype
uint16[] position
uint16 distance
uint16 angle
bool angle_enslavment
uint16 max_speed_or_soft_hard